src/pineko/theory.py

```diff
@@ -389,49 +385,38 @@ def fk(self, name, grid_path, tcard, pdf):
                 raise ValueError(
                     "Factorization scale variations are not available for this grid"
                 )
-        # collect alpha_s
-        # TODO: move this down to evolve.evolve_grid when output contains cards
-        new_tcard = eko.compatibility.update_theory(tcard)
-        astrong = sc.Couplings.from_dict(new_tcard)
-        # ocard = self.load_operator_card(name)
-        # q2_grid = ocard["Q2grid"]
-
         # loading ekos
-        operators = eko.output.legacy.load_tar(eko_filename)
-        muf2_grid = operators.Q2grid
-        # PineAPPL wants alpha_s = 4*pi*a_s
-        # remember that we already accounted for xif in the opcard generation
-        alphas_values = [
-            4.0 * np.pi * astrong.a_s(xir * xir * muf2 / xif / xif)
-            for muf2 in muf2_grid
-        ]
-        # Obtain the assumptions hash
-        assumptions = theory_card.construct_assumptions(tcard)
-        # do it!
-        logger.info("Start computation of %s", name)
-        logger.info("max_as=%d, max_al=%d, xir=%f, xif=%f", max_as, max_al, xir, xif)
-        start_time = time.perf_counter()
+        with eko.EKO.edit(eko_filename) as operators:
```

Comment:
Actually, why do we need `edit` here? can't we use just `open` and rotate every time? (Remember that `edit` allows to write back to disk)